Libraries/Text/TextInput/RCTBackedTextInputDelegateAdapter.m

```diff
@@ -107,6 +107,12 @@ - (BOOL)textFieldShouldReturn:(__unused UITextField *)textField
 
 - (void)textFieldDidChange
 {
+  UITextRange *markedTextRange = _backedTextInputView.markedTextRange;
+
+  if (markedTextRange != nil && !markedTextRange.empty) {
```

Comment:
Nit: the getter's name is [`isEmpty`](https://developer.apple.com/documentation/uikit/uitextrange/1614556-empty?language=objc)

Reply:
Thanks